app/models/question_set.rb

```diff
@@ -1,25 +1,28 @@
 class QuestionSet < ApplicationRecord
   include Translatable
 
-  has_closure_tree_root :root_group, class_name: "Question"
+  KINDS = %i[loan_criteria loan_post_analysis].freeze
+
+  belongs_to :division, inverse_of: :question_sets
+  has_many :response_sets, inverse_of: :question_set, dependent: :restrict_with_exception
 
-  translates :label
```

Comment:
This was not getting used it seems. We may end up adding it back later if QuestionSets become editable but for now it's unnecessary.

Reply:
thanks for taking this out